src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperDescriptorFactory.cs

```diff
@@ -246,7 +269,8 @@ internal static IEnumerable<string> GetCommaSeparatedValues(string text)
                         parentType: type,
                         errorSink: errorSink,
                         defaultPrefix: descriptor.Name + "-",
-                        isInvalid: out isInvalid);
+                        isInvalid: out isInvalid,
+                        designTime: designTime);
```

Comment:
leave `out` parameters last. less important but would also be good to place `designTime` consistently just before (or after) `errorSink`.